src/TwigComponent/src/Twig/TwigPreLexer.php

```diff
@@ -201,6 +203,15 @@ private function consumeAttributes(string $componentName): string
                 break;
             }
 
+            if ($this->check('{{...') || $this->check('{{ ...')) {
```

Comment:
I think you should consume the whitespace before doing this check, tab, or multiples tab can be valid syntax too.